code/src/cljs/sixsq/nuvla/ui/profile/views.cljs

```diff
@@ -1138,8 +1138,9 @@
            [ui/TableRow
             [ui/TableCell [:b (@tr [:street-address])]]
             [ui/TableCell street-address [:br] postal-code " - " city [:br] country]]
-           ]]
-         ]))))
+          [ui/TableRow
+           [ui/TableCell {:width 5} [:b (str/capitalize (@tr [:credit-balance]))]]
+           [ui/TableCell {:width 11} (format-currency currency balance)]]]]]))))
```

Comment:
Balance can be positive or negative. I think we can add a green color when there is credit and red when an amount is owed.